docs/HowTo/clusterconnectivity/aiven-karapace-cluster-ssl-protocol.md

```diff
@@ -44,14 +44,14 @@ registry cluster with Klaw using SSL protocol:
 6. In the **Schema Registry Environments** section, click **Add
    Environment** and enter the details to add your schema registry
    environment. Click **Save**.
-7. Open the `application.properties` file for [cluster-api]
-   (klaw/cluster-api/src/main/resources) modules.
+7. Open the `application.properties` file for [cluster-API]
```

Comment:
This occurrence should not be capitalised, as it's part of the URL. It should be replaced with `cluster-api` (in backticks) as it indicated a directory name.